manager/src/test/resources/application.properties

```diff
@@ -1,5 +1,5 @@
 # Dev properties
-%dev.quarkus.log.category."org.hibernate".min-level=DEBUG
+quarkus.log.category."org.hibernate".min-level=DEBUG
```

Comment:
Is this needed? Wouldn't it spam test logs?

Reply:
thank you @sutaakar , it was a leftover

Reply:
It was in the tests btw

Reply:
@sutaakar it is in the tests. Since it was in the test `dev` is never active so it was not used at all. If we want to remove such logs from the tests we can remove the line entirely

Reply:
I think it could be removed